dd-sdk-android/src/main/kotlin/com/datadog/android/core/internal/utils/RuntimeUtils.kt

```diff
@@ -47,15 +49,15 @@ internal fun buildSdkLogger(): Logger {
  */
 internal val devLogger: Logger = buildDevLogger()
 
-private fun buildDevLogger(): Logger {
-    return Logger(buildDevLogHandler())
+private fun buildDevLogger(sdkCore: SDKCore? = null): Logger {
```

Comment:
In which case we may want this method to be called without argument or with `null` value? Or this is just for the transition period?

Reply:
This is just for the transition period.